### DISCUSSION TITLE
Apex generator not available in Brew Install ?

### DISCUSSION THREAD

**Discussion by sfdev99** [swagger-api/swagger-codegen#10955] [Q&A]
Hi, using Brew, I installed the swagger-codegen 3.0.25 on a Mac OS X. It runs fine for ruby but it says the apex config file is not available.

java.lang.RuntimeException: Can't load config class with name apex

What's the best way to add the apex generator to my set up ? Any help appreciated. 

thks

**Comment by vctqs1** [comment]
I get same isssue.
